dapps/addressbook.vue/src/i18n/de.ts

```diff
@@ -44,8 +45,8 @@ export default{
         "title": "Kontaktanfrage"
       },
       "continue": "Weiter",
-      "desc": "Über dieses Formular können Sie einen Kontakt über eine bestehende evan.network Identität oder per E-Mail einladen. Sobald Sie einen Kontakt eingeladen haben, können Sie mit diesem sichere Nachrichten austauschen und ihn in Verträge einladen.",
-      "desc-email": "Sobald ein Nutzer per E-Mail eingeladen wird, können Sie diesen Kontakt nicht direkt verwenden, um mit diesem Daten zu teilen. Sobald der Anmeldeprozess des eingeladenen Benutzers abgeschlossen ist, erhalten Sie eine Blockchain Mail mit der Sie den Einladeprozess beenden und Kontaktdaten aktualisieren können. Weiterhin können Sie EVE als Startguthaben mitschicken.",
+      "desc": "Über dieses Formular können Sie einen Kontakt über eine bestehende evan.network Identität oder per E-Mail einladen. Sobald Sie einen Kontakt eingeladen haben, können Sie mit diesem Informationen austauschen.",
+      "desc-email": "Sie können hier einen weiteren Partner auf Basis der E-Mail einladen und ihm EVE für eine erste Interaktion im evan.network senden. Sobald sich Ihr Partner im evan.network angemeldet hat, erhalten Sie eine Information in Ihrem Account.",
```

Comment:
"Benachrichtung" instead of "Information"?

```suggestion
      "desc-email": "Sie können hier einen weiteren Partner auf Basis der E-Mail einladen und ihm EVE für eine erste Interaktion im evan.network senden. Sobald sich Ihr Partner im evan.network angemeldet hat, erhalten Sie eine Benachrichtigung in Ihrem Account.",
```